### PR TITLE
chore: Migrate views

### PR DESCRIPTION
**Pull request by RabbitDoge**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,40 +33,40 @@
   },
   "dependencies": {
     "@binance-chain/bsc-connector": "^1.0.0",
-    "@ethersproject/experimental": "^5.4.0",
+    "@ethersproject/experimental": "^5.0.1",
```
Comment (Chef-Cheems) [inline package.json:36]: Why downgrading some packages?

File: public/manifest.json [hunk 2]
```diff
@@ -2,15 +2,22 @@
   "short_name": "PancakeSwap",
   "name": "PancakeSwap",
   "description": "Farm CAKE with Pancake LP Tokens",
+  "homepage_url": "https://exchange.pancakeswap.finance/#/",
```
Comment (Chef-Cheems) [inline public/manifest.json:5]: Should be changed to `https://pancakeswap.finance` I guess

File: src/components/Nav/Menu/components/Accordion.tsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+const AccordionContent = styled.div<{ isOpen: boolean; isPushed: boolean; maxHeight: number }>`
+  max-height: ${({ isOpen, maxHeight }) => (isOpen ? `${maxHeight}px` : 0)};
+  transition: max-height 0.3s ease-out;
+  overflow: hidden;
+  border-color: ${({ isOpen, isPushed }) => (isOpen && isPushed ? 'rgba(133, 133, 133, 0.1)' : 'transparent')};
+  border-style: solid;
+  border-width: 1px 0;
+`
+
+const Accordion: React.FC<Props> = ({
```
Comment (Chef-Cheems) [inline src/components/Nav/Menu/components/Accordion.tsx:34]: This component is already present in UIKit

File: src/components/Nav/Menu/components/Avatar.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+const StyledAvatar = styled.div`
+  margin-left: 8px;
+  position: relative;
+
+  img {
+    border-radius: 50%;
+  }
+`
+
+const Avatar: React.FC = () => {
```
Comment (Chef-Cheems) [inline src/components/Nav/Menu/components/Avatar.tsx:16]: This component is already in UIKit (although its a bit more complicated over there)

File: src/components/Nav/Menu/components/CakePrice.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    transition: transform 0.3s;
+  }
+  :hover {
+    svg {
+      transform: scale(1.2);
+    }
+  }
+`
+
+const CakePrice: React.FC = () => {
```
Comment (Chef-Cheems) [inline src/components/Nav/Menu/components/CakePrice.tsx:19]: This component is already in UIKit

File: src/components/Nav/Menu/components/LangSelector.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Dropdown, Text, Button, LanguageIcon } from '@pancakeswap/uikit'
+import { languageList } from 'config/localization/languages'
+import { useTranslation } from 'contexts/Localization'
+import MenuButton from './MenuButton'
+
+const LangSelector: React.FC = () => {
```
Comment (Chef-Cheems) [inline src/components/Nav/Menu/components/LangSelector.tsx:7]: Same as above

File: package.json (hunk 1 above)
Comment (RabbitDoge) [reply package.json:36]: not working otherwise

**Pull request merged**